examples/gno.land/p/demo/blog/util.gno

```diff
@@ -1,7 +1,19 @@
 package blog
 
-import "strings"
+import (
+	"strconv"
+	"strings"
+	"time"
+)
 
 func breadcrumb(parts []string) string {
 	return "# " + strings.Join(parts, " / ") + "\n\n"
 }
+
+func makePubDate(t time.Time, id int64) string {
+	// We use the timestamp to sort by publish order.
+	// In case of multiple msgs to publish in the same transaction
+	// there will be a timestamp conflict (time.Now() is per transaction, not per msg).
+	// This functions assures that timestamp IDs are unique in this case.
+	return strconv.Itoa(int(t.Unix())) + "-" + strconv.Itoa(int(id))
```

Comment:
It's a bit of an edgecase. But consider padding these with zeroes if you wanna use them in avl. Otherwise you get hung up in the same old problem of "9" > "120000".

Reply:
I removed this function and changed the approach totally, please look at the last commit :)